### ISSUE TITLE
Useless line?

### ISSUE THREAD

**Issue by FrankBro** [RAttab/rill#5]
https://github.com/RAttab/rill/blob/master/src/query.c#L49

**Comment by RAttab** [comment]
Yes and no. It's there so I don't forget about it if I ever end up adding extra gotos for cleanup. The compiler should be able to treat it as dead code and remove it during compilation.
